packages/rpc-core/src/rpc-methods/getBlockTime.ts

```diff
@@ -0,0 +1,14 @@
+import { U64UnsafeBeyond2Pow53Minus1 } from './common';
+
+/** Estimated production time, as Unix timestamp (seconds since the Unix epoch) */
+type GetBlockTimeApiResponse = U64UnsafeBeyond2Pow53Minus1;
```

Comment:
I'm _sort_ of inclined to make a new opaque type that represents timestamps. What do you think?

Something like this:

https://github.com/solana-labs/solana-web3.js/blob/d16acbdd1d451120ddac983e076545ce35934785/packages/rpc-core/src/rpc-methods/common.ts#L12

…but like this:

```ts
type UnixTimestamp = number & {readonly __unixTimestamp: unique symbol};
```

Reply:
I like this idea! The only downside is that we patch a `u64` response from the RPC to number, which is maybe a bit confusing despite being safe (until 2255). I think it makes sense though, updated to use this

Reply:
It's actually more complicated that, even. JavaScript date maxes out at:

```ts
new Date(8.64e15)
> Fri Sep 12 275760 17:00:00 GMT-0700 (Pacific Daylight Time)
```

Which is lower even than `Number.MAX_SAFE_INTEGER`.

Anyway, see you in the year 275760.